dagshub/auth/oauth.py

```diff
@@ -31,8 +32,10 @@ def oauth_flow(host: str, client_id: Optional[str] = None) -> OAuthDagshubToken:
     state = uuid.uuid4()
     middle_man_request_id = hashlib.sha256(uuid.uuid4().bytes).hexdigest()
     auth_link = (
-        f"{dagshub_url}/authorize?state={state}&client_id={client_id}" f"&middleman_request_id={middle_man_request_id}"
+        f"{dagshub_url}/authorize?state={state}&client_id={client_id}&middleman_request_id={middle_man_request_id}"
```

Comment:
1. How did this even work\compile previously?
2. I think you can get rid of the surrounding parens now

Reply:
It was an artifact of the formatter, but the syntax is valid Python and it just sticks them together
you can try this in a REPL:
```
>>> a = "aaa" "bbb"
>>> a
'aaabbb'
```

What happened here is at some point the formatter split it into two strings, but then later when the line length got raised to 120, instead of sticking them together, it just put them on a single line.

The parens I can't get rid of, putting the string inline with auth_link goes over the line length, so the formatter stuck in the parens